src/plugins/input_control_vis/public/input_control_vis_type.ts

```diff
@@ -37,8 +36,9 @@ export function createInputControlVisTypeDefinition(
       defaultMessage: 'Controls',
     }),
     icon: 'controlsHorizontal',
+    group: VisGroups.TOOLS,
     description: i18n.translate('inputControl.register.controlsDescription', {
-      defaultMessage: 'Create interactive controls for easy dashboard manipulation.',
+      defaultMessage: 'Create controls to interact with your dashboard.',
```

Comment:
Since you changed the string here, this translation needs to be removed from all existing translation files, so it will get a new translation in the next translation cycle.

Reply:
Oh these translations again 😄 Thanx Tim will do!